lib/DataDog/DogStatsd/Helper.pm

```diff
@@ -65,7 +65,11 @@ sub stats_event      { get_dogstatsd()->event(@_); }
 
 my $DOGSTATSD;
 sub get_dogstatsd {
-    $DOGSTATSD ||= DataDog::DogStatsd->new;
+    $DOGSTATSD ||= DataDog::DogStatsd->new(
+        host        => $ENV{DATADOG_HOST},
+        port        => $ENV{DATADOG_PORT},
+        namespace   => $ENV{DATADOG_NAMESPACE}
```

Comment:
The environment variables need a default (as in the case where these are *not* set.)

Also likely needs tests on the `DataDog::DogStatsd` side for these attribute settings.

Reply:
The default values are handled in `DataDog::DogStatsd` contractor, so even if we keep it undefined this will not be a problem

Reply:
Then it'd make more sense if the environment variables are handled on `DataDog::DogStatsd` in case anyone else is using that and not this helper.

Reply:
If he is using `DataDog::DogStatsd` he can overwrite the `host` and the `port` bypassing them to the contractor when he creates a new object from `DataDog::DogStatsd` class but in the helper, we are creating the object internally